packages/marshal/marshal.js

```diff
@@ -20,24 +37,20 @@ export function getInterfaceOf(maybeRemotable) {
  * The resulting copy is guaranteed to be pure data, as well as hardened.
  * Such a hardened, pure copy cannot be used as a communications path.
  *
- * @template T
- * @param {T} val input value.  NOTE: Will become hardened!
+ * @template {Error|{}} T
```

Comment:
Why does this mention `Error`? Is it some typescript type magic special case? Also, doesn't this imply that the argument must be an object? In fact, the argument can also be a primitive value, and is a string in the one case we use.

Reply:
Yes, that was a mistake.  I updated to use a different dance around the type system in the `Error` case.